frontend/src/components/TimeLogsPage/TimeLogRow.jsx

```diff
@@ -10,34 +10,44 @@ export const TimeLogRow = ({ log, handleDelete }) => {
   const { hours, minutes } = minutesToFormattedHoursAndMinutes(log.minutes)
   const [confirmOpen, setConfirmOpen] = useState(false)
 
+  const dateObj = new Date(log.date)
+  const formattedDate = `${dateObj.getDate().toString().padStart(2, '0')}/${(dateObj.getMonth() + 1).toString().padStart(2, '0')}/${dateObj.getFullYear()}`
```

Comment:
You could make this a bit simpler by using:

`const formattedDate = new Intl.DateTimeFormat('fi-FI').format(dateObj)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat